### ISSUE TITLE
Shortcuts are not created after restarting the phone

### ISSUE THREAD

**Issue by antodc** [FossifyOrg/Phone#268]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Phone/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Phone/issues) or [closed](https://github.com/FossifyOrg/Phone/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Phone/discussions) or [closed](https://github.com/FossifyOrg/Phone/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Phone/blob/master/CONTRIBUTING.md).

### Affected app version

1.3.0

### Affected Android/Custom ROM version

Android 11

### Affected device model

Oscal C60

### How did you install the app?

GitHub releases

### Steps to reproduce the bug

1 Restart the phone
2 Open Fossify Phone
3 Go to contact list
4 Select one contact
5 Select “create shortcut” from three dots menu

### Expected behavior

A shortcut should appear in the home page

### Actual behavior

Nothing happens 

### Additional information

If one changes the launcher/home app then the shortcuts are created, but once the phone is restarted they are not created anymore.
I tried two launchers (Fossify Home and Discreet Launcher), so I think the problem is with Fossify Phone.

**Comment by Aga-C** [comment]
Does adding shortcuts work in other apps, especially in non-Fossify apps?

**Comment by antodc** [comment]
I tried to create shortcuts to files with Fossify File Manager and Total Commander. None worked. Then, I switched to the stock launcher and everything works (Fossify Phone and File Manager and Total Commander), also after restarting the phone.
With the stock launcher the management of the shortcuts is different from the other two. It asks where to put the shortcuts in the home screen, while both Fossify Home and Discreet Launcher try to create them directly. Maybe that can explain something?
